README.md

```diff
@@ -11,14 +11,14 @@ An [Apache Parquet](https://parquet.apache.org/) implementation in Rust (work in
 - Rust nightly
 - Thrift 0.11+ or master (with Rust support)
 
-To install Rust nightly build, just run (assuming you have Rust installed already):
+To install Rust nightly build, run this (assuming you have Rust installed already):
 ```shell
 rustup update nightly
 rustup default nightly
 ```
 
 Project requires Thrift with Rust support, which is in 0.11+/master. If release is not available on
```

Comment:
I think we can change "0.11+/master" to "master": I'm not sure if a 0.11 release will ever happen. And we can change this after the next version is released.

Reply:
Okay, will change.